tamr_toolbox/utils/downstream.py

```diff
@@ -0,0 +1,245 @@
+import logging
+from tamr_unify_client.dataset.resource import Dataset
+from tamr_unify_client.project.resource import Project
+from tamr_unify_client.client import Client
+from typing import List, Optional, Dict
+from collections import OrderedDict
+from tamr_toolbox.models.project_type import ProjectType
+import re
+
+LOGGER = logging.getLogger(__name__)
+
+
+def datasets(dataset: Dataset, *, include_dependencies_by_name: bool = False) -> List[Dataset]:
+    """Returns a dataset's downstream datasets.
+
+    Args:
+        dataset: The target dataset.
+        include_dependencies_by_name: A Boolean indicating whether to return datasets with
+            dependencies suggested by name matching. Default to False.
```

Comment:
Avoid mentioning data types and default values in the DocString where possible as they are already part of the function definition (have having the information in two places allows them to conflict):
Whether to include datasets based on name similarity